ee/src/main/java/org/jboss/as/ee/subsystem/EeSubsystemAdd.java

```diff
@@ -211,5 +213,15 @@ protected void execute(DeploymentProcessorTarget processorTarget) {
 
         context.getServiceTarget().addService(ReflectiveClassIntrospector.SERVICE_NAME, new ReflectiveClassIntrospector()).install();
 
+        reportEEPreviewMode();
+    }
+
+    private void reportEEPreviewMode() {
+        String previewMode = WildFlySecurityManager.getPropertyPrivileged("ee8.preview.mode", null);
+        if ("true".equals(previewMode)) {
```

Comment:
Should this be `"true".equalsIgnoreCase(previewMode)`? Or simply it could just be `if (Boolean.parseBoolean(previewMode))`.

Reply:
The latter looks sleek.

Reply:
This is a question for @dmlloyd. Basically this should do the same thing that JBoss Modules does.  I wrote ^^^ just by looking at the module.xml text and writing Java that said the same thing as the text. :)

Thanks for asking; I was in a hurry when I wrote this and didn't take the time to research.

Reply:
I believe that the condition is doing an exact equality check.